internal/locale/locales/en-us.yaml

```diff
@@ -2080,5 +2080,10 @@ err_local_commit_file:
     To view your projects commits run '[ACTIONABLE]state history[/RESET]'.
     
     Please avoid deleting or editing this file directly.
+projectmigration_confirm:
+  other: |
+    Your project is currently read-only until you migrate to the new project format.
+    After you migrate, older versions of the State Tool will not be able to use your project until they update.
+    Would you like to perform the migration now?
```

Comment:
@rawktron please review

Reply:
What's the full message? We have to give them some context for this. Is there another part of this message?

Reply:
It should be something like "State Tool has introduced a new project format to provide enhanced functionality. Your project is currently using the old format and will remain read-only until you migrate to the new project format.

WARNING: After you migrate, older versions of the State Tool will not be able to use your project.

Would you like to perform the migration now?"